.github/workflows/docker/compose/llms-compose.yaml

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # this file should be run in the root of the repo
-# images used by GenAIExamples: llm-tgi,llm-ollama,llm-docsum-tgi,llm-faqgen-tgi,llm-vllm,llm-vllm-hpu,llm-vllm-ray,llm-vllm-ray-hpu
+# images used by GenAIExamples: llm-tgi,llm-ollama,llm-docsum-tgi,llm-faqgen-tgi,llm-vllm,vllm-hpu,llm-vllm-ray,llm-vllm-ray-hpu
```

Comment:
It seems some other image names need to be updated as well e.g., llm-vllm-ray-hpu. @yongfengdu are you going to fix as well?

Reply:
Sure, let me check and fix them together